relayer/workers/parachaincommitmentrelayer/catchup.go

```diff
@@ -183,7 +206,7 @@ func (li *BeefyListener) parablocksWithProofs(blocks []ParaBlockWithDigest, late
 			MMRProofResponse: mmrProof,
 			Header:           ownParaHead,
 			HeaderProof:      ownParaHeadProof,
-			HeaderProofPos:   ownParaHeadProofPos,
+			HeaderProofPos:   0,
```

Comment:
@musnit, The index of our parachain header in the response for the `Paras.Heads` storage query likely isn't relevant for any purpose involving the Merkle Tree of Para Heads. They are two different trees. So I've just set this to 0 for now.

Reply:
@vgeddes I think we need the index in order to verify the merkle proof: https://github.com/Snowfork/snowbridge/blob/main/ethereum/contracts/utils/MerkleProof.sol#L34

Reply:
The `Paras.Head` storage map is part of the base-16 modified merkle-patricia tree that is used for storage on all substrate chains. That's surely different from the plain binary Merkle tree as encoded in the MMR?

Reply:
so afaik the plain binary tree is created from all parachain heads based on the order that the Paras.Heads query returns, so that query returns the leaves which are put into the binary tree. the position in the array there should correspond to the leaf position in the binary merkle tree

Reply:
I see. I've updated the code as necessary.